integration-tests/multi_contract_ref/README.md

```diff
@@ -1,13 +1,14 @@
-# Delegator Smart Contract
+# `MultiContractRef` Smart Contract
```

Comment:
The name can be confused with the one automatically generated by `ContractRef`.
I would name it smth like 
```suggestion
# `MultiContractCaller` Smart Contract
```
or just
```suggestion
# `MultiContract` Smart Contract
```